test/restart/mzcompose.py

```diff
@@ -158,23 +156,6 @@ def workflow_stash(c: Composition) -> None:
 
         cursor.execute("CREATE TABLE b (i INT)")
 
-        c.rm("cockroach")
-        c.up("cockroach")
-
-        # CockroachDB cleared its database, so this should fail.
-        #
-        # Depending on timing, this can fail in one of two ways. The stash error
-        # comes from the stash complaining. The network error comes from pg8000
-        # complaining because Materialize panicked.
-        try:
-            # Reusing the existing connection means we don't need to worry about
-            # detecting `ConnectionRefused` errors
-            cursor.execute("CREATE TABLE c (i INT)")
-            raise Exception("expected unreachable")
-        except pg8000.exceptions.InterfaceError as e:
-            if str(e) != "network error":
-                raise e
-
```

Comment:
This has gotten really annoying to debug and in my opinion doesn't really test anything useful, so I just deleted it.